src/Executor/Executor.php

```diff
@@ -163,7 +163,7 @@ public static function promiseToExecute(
      *
      * @param mixed        $source
      * @param mixed[]      $args
-     * @param mixed[]|null $context
+     * @param mixed|null   $context
```

Comment:
you have 2 more spaces here

Reply:
done, I have readjusted the whole param block